pkgs/development/libraries/lensfun/default.nix

```diff
@@ -1,24 +1,43 @@
-{ stdenv, fetchurl, pkgconfig, glib, zlib, libpng, cmake }:
-
-stdenv.mkDerivation rec {
+{ stdenv, fetchFromGitHub, pkgconfig, glib, zlib, libpng, cmake }:
+let
```

Comment:
```suggestion
{ stdenv, fetchFromGitHub, pkgconfig, glib, zlib, libpng, cmake }:

let
```

Reply:
`nixpkgs-fmt` disagrees with that line ;-)

Reply:
There is always a new line between }: and the mkDerivation function. Styling is not defined by nixpkgs-fmt but by the contribution guide. nixpkgs-fmt could do whatever and it wouldn't be right.

Reply:
fine, I updated it manually, but as my editor does nixpkgs-fmt a lot of stuff automatically, it's good to get this fixed. I commented at https://github.com/nix-community/nixpkgs-fmt/issues/218#issuecomment-758693665.